--- J2735-Common-2024-rel-v1.1.2.asn	2024-05-16 14:48:14.000000000 -0600
+++ J2735-Common-2024-rel-v1.1.2.asn.EDITED	2025-01-17 16:10:23.262921200 -0700
@@ -1874,11 +1874,11 @@
    eventWipersChanged              (9),
    eventFlatTire                   (10),
    eventDisabledVehicle            (11), -- The DisabledVehicle DF may also be sent
-   eventAirBagDeployment           (12),
-   eventJackKnife                  (13)  -- Applies to vehicles with trailer(s)
-   } (SIZE (13, ..., 14))
+   eventAirBagDeployment           (12)
+   } (SIZE (13, ...))
 -- To extend: Append event flags to the list,
 -- and update extension marker, e.g., (Size (13, ..., 14..16))
+-- EDITED: reverted to previous definition without 'eventJackKnife'
 
 VehicleHeight ::= INTEGER (0..127) 
     -- the height of the vehicle
