--- J2945-3-RoadWeatherMessage-2024-rel-v2.1.asn	2024-03-25 13:58:52.000000000 -0600
+++ J2945-3-RoadWeatherMessage-2024-rel-v2.1.asn.EDITED	2024-11-17 00:22:05.128855100 -0700
@@ -136 +136 @@
-    citizenReport   SnapShot,
+    citizenReport   RwmSnapShot,    -- EDITED
@@ -141 +141 @@
-    snapShot         SnapShot,
+    snapShot         RwmSnapShot,   -- EDITED
@@ -222,2 +222,4 @@
-
-SnapShot ::= SEQUENCE {
+-- EDITED
+-- Name changed to avoid conflict with "Snapshot", lowercase S, in module ProveVehicleData per
+-- this https://github.com/vlm/asn1c/issues/420
+RwmSnapShot ::= SEQUENCE {
