--- J3217-TollUsageMsg-2024-rel-v1.1.asn	2024-02-21 17:09:52.000000000 -0700
+++ J3217-TollUsageMsg-2024-rel-v1.1.asn.EDITED	2024-11-17 00:18:53.773971300 -0700
@@ -157 +157 @@
-    vehicleId           VehicleId, 
+    vehicleId           TumVehicleId, 
@@ -184 +184,3 @@
-VehicleId ::= SEQUENCE {
+-- EDITED to avoid conflict with "VehicleID" in "Common" module.
+-- Name changed VehicleId -> TumVehicleId
+TumVehicleId ::= SEQUENCE {
